src/impl/util.ts

```diff
@@ -52,65 +54,68 @@ export function hasRelative() {
 
 // OBJECTS AND ARRAYS
 
-export function maybeArray(thing) {
+export function maybeArray(thing: unknown) {
```

Comment:
```ts
export function maybeArray<T>(thing: T | ReadonlyArray<T>): T[] {
```
So the returned value is auto-typed based on the in the input.

Reply:
Well spotted.